pkgs/development/libraries/libserdes/default.nix

```diff
@@ -39,6 +39,9 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     patchShebangs configure lds-gen.pl
+  '' + lib.optionalString (stdenv.cc.libcxx != null) ''
+    substituteInPlace src-cpp/Makefile \
+      --replace "LIBS += -lstdc++" "LIBS += -lc++ -l${stdenv.cc.libcxx.cxxabi.libName}"
```

Comment:
I see something like this in a [few places in nixpkgs](https://github.com/NixOS/nixpkgs/blob/47a0bcedb1f12363480ffcf5f923f95fb350a068/pkgs/by-name/ni/nickel/package.nix#L40-L43). Is the cause same, #166205?

Would the same `NIX_LDFLAGS` workaround be valid here?

Anyway, if this indeed is related to #166205, a comment referring that issue would be nice.

Reply:
Do you think `stdenv.cc.isClang` would be a better condition to use then, instead of the null check?